skimage/measure/mc_meta/createluts.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+
+""" Create lookup tables for the marching cubes algorithm, by parsing
+the file "LookUpTable.h". This prints a text to the stdout wich
+can then be copied to luts.py.
+
+The luts are tuples of shape and base64 encoded bytes.
+
+"""
+  
+import numpy as np
+import sys
+import base64
+
+# Get base64 encode/decode functions
+PY3 = sys.version_info[0] == 3
+if PY3:
```

Comment:
Oh crap, I never write it this way anymore, but I suppose I did back when I wrote this :/